package.json

```diff
@@ -151,32 +134,49 @@
     "chai-shallow-deep-equal": "^1.4.6",
     "chance": "^1.0.16",
     "cheerio": "^1.0.0-rc.2",
+    "chokidar": "^2.0.3",
     "co": "~4.6.0",
-    "enzyme": "^3.6.0",
-    "enzyme-adapter-react-16": "^1.5.0",
+    "clean-webpack-plugin": "^0.1.19",
+    "css-loader": "^1.0.0",
+    "cssnano": "^4.0.0",
+    "enzyme": "^3.5.1",
+    "enzyme-adapter-react-16": "^1.4.0",
     "enzyme-to-json": "^3.3.4",
     "eslint": "^5.0.0",
     "eslint-plugin-import": "^2.12.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-mocha": "^5.0.0",
     "eslint-plugin-react": "^7.9.1",
-    "fetch-mock": "^7.0.2",
+    "fetch-mock": "^6.4.4",
```

Comment:
from 7.x to 6.x

Reply:
these are both back up to latest in master now